web/themes/custom/novel/templates/views/input--search-api-fulltext.html.twig

```diff
@@ -0,0 +1,3 @@
+{# web/core/themes/stable9/templates/form/input.html.twig #}
+<input{{ attributes.addClass('search-full-text__input') }} />
+<input type="image" src="/themes/custom/novel/assets/dpl-design-system/icons/basic/icon-search.svg" alt="Søgeknap">
```

Comment:
```suggestion
<input type="image" src="/themes/custom/novel/assets/dpl-design-system/icons/basic/icon-search.svg" alt="{{ 'Search button'|trans }}">
```

Reply:
Neither "Søgeknap" nor "Search button" is correct. The alt text should be the alternative representation if the image isn't used, so it should be "Search".